pages/cockpit/overview/introduction.adoc

```diff
@@ -7,37 +7,51 @@
+* *Environment*: at the third level, an environment in an IT infrastructure, such as development or production. There can be multiple environments linked to one organization.
+* APIM or AM *Installations*, linked to environments in Cockpit.
+* Each linked installation automatically reports its *Nodes* to Cockpit.
++
+Learn more about <<nodes>>
 
-You can link your APIM or AM _installations_ at the environment level in Cockpit. Each registered installation will then automatically reports its _nodes_ to Cockpit (a node being either a management-api node or a gateway node).
+image::cockpit/entity-hierarchy.png[]
 
-This hierarchy is represented in Cockpit by a visual map on the dashboard.
+The Cockpit hierarchy is represented visually in Cockpit in the hierarchy map pictured at the top of the page.
 
-.Hierarchy of a Cockpit account
-image::cockpit/entity-hierarchy.png[]
+=== Entity properties
 
+Each entity managed in Cockpit has some common properties:
 
-NOTE: The organization and environment entities defined in Cockpit are equivalent to the same entities in APIM and AM, including the roles you can define for them (for example, the `ORGNIZATION_OWNER` role exists in both APIM and AM). For more information, see the APIM and AM product documentation.
+* ID: an internal ID that is never shown in the Cockpit UI, but that you can find if you look at the API responses.
+* HRID: a human readable ID of the entity. This ID is unique (no two environments in the same organization can have the same HRID), and they are used to provide readable URLs.
+* Name: the name of the entity.
+* Description: a description of the entity.
 
-== Cockpit entities
+== Nodes
 
-Each entity managed in Cockpit have some common properties:
+APIM and AM installations report on one or more instances of the following node types to Cockpit:
 
-* ID: a technical ID that is never shown on the UI, but that you can find if you look at the API responses
-* HRID: a human readable ID of the entity. This ID is unique (2 environments of the same organization can not have the same HRID), and they are used to provide readable URLs.
-* Name: the name of the entity
-* Description: a description of the entity.
+- REST API nodes
+- Gateway nodes
+
+The Cockpit hierarchy pictured below has the following setup:
+
+- One APIM installation, with two Gateway nodes and one REST API node.
+- One AM installation, with one Gateway node and one REST API node.
+
+image::cockpit/apim-am-nodes.png[]
+
+The organization and environment entities defined in Cockpit are equivalent to the same entities in APIM and AM, including the roles you can define for them (for example, the `ORGNIZATION_OWNER` role exists in both APIM and AM).
+All nodes can belong to multiple environments, but only the Gateway node organization and environments are configurable in APIM and AM.
```

Comment:
Not sure I understand this sentence... Is this rephrasing correct ? 
> All nodes can belong to multiple environments. You can restrict which organization and environments are handled on the Gateway nodes in APIM and AM.

And maybe if it is, we could add an indication of how it can be configured (menu entry path or screenshot) ?

Reply:
Ilies will give more detail about the APIM/AM config scenario.